.gitignore

```diff
@@ -2,10 +2,9 @@
 /.gitconfig
 
 /.cache
-/.idea
+.idea
 /.m2
 
-code/packaging/app/keys.properties
```

Comment:
Is there a specific reason you removed this file?  That file gets generated a lot of the times when you launch the application through Eclipse and developers kept putting it in PRs, so we ignore it.